src/Components/Molecules/Caption/index.tsx

```diff
@@ -0,0 +1,19 @@
+import { ReactNode } from 'react'
+import Text from '../../Atoms/Text'
+import style from './Caption.module.scss'
+
+type CaptionType = {
+    icon: ReactNode;
+    label: string;
+    isLink?: boolean;
+    onClick?: VoidFunction;
```

Comment:
Eu deixaria aqui obrigatorio, onClicks pode sempre deixar obrigatorios
```suggestion
    onClick: VoidFunction;
```